build/main.js

```diff
@@ -16,84 +16,29 @@
 require('babel-polyfill');
 require('babel-register')();
 
+const Tasks = require('./tasks');
+const Node = require('./utils/node');
```

Comment:
Maybe `requirements` or `runtime` would be a better name for this module -- its unclear what it does by the name